source/features/highlight-user-prs.tsx

```diff
@@ -0,0 +1,18 @@
+import select from 'select-dom';
+import features from '../libs/features';
+import {getUsername} from '../libs/utils';
+
+function init() {
+	for (const openedByEl of select.all('span.opened-by > a')) {
+		if (openedByEl.textContent === getUsername()) {
```

Comment:
We can skip the `if` if you select by attribute. I think it would be something like 

```js
select.all(`.opened-by [href$="/${CSS.escape(getUsername())}"]`)